package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "build": "yarn bundle",
     "bundle": "NODE_ENV=production yarn webpack --color --mode=production --config config/webpack.config.prod.js",
-    "start": "GRAPHQL_SECRET=d8Ed.-qfhj7 standard && node scripts/start.js",
```

Comment:
this seems wrong

Reply:
Yep, the variable should go before `node`:

```bash
standard && GRAPHQL_SECRET=d8Ed.-qfhj7 node scripts/start.js
```

This secret is just for the dev environment and it's the same as in the backend config. For the build we need to set this variable manually:

```fish
$ set GRAPHQL_SECRET=...
$ yarn build
```